Controls/Grid/Grid.UWP/View/Services/Editing/EditingService.cs

```diff
@@ -118,18 +118,20 @@ public bool CommitEdit(ActionTrigger trigger)
                         if (trigger != ActionTrigger.ExternalEditor)
                         {
                             this.Owner.ExternalEditor.CommitEdit();
+                            this.Owner.CurrencyService.RefreshCurrentItem(true);
                         }
                     }
                     else
                     {
                         this.Owner.Model.CommitEdit();
                         this.Owner.EditRowLayer.EditorLayoutSlots.Clear();
+                        this.Owner.CurrencyService.RefreshCurrentItem(true);
                     }
 
                     this.Owner.Model.CurrentDataProvider.CommitEditOperation(this.EditItem);
                 })
                 {
-                    Flags = UpdateFlags.AffectsContent,
+                    Flags = UpdateFlags.AffectsData,
```

Comment:
Why is this change needed? AffectsData has a lower priority compared to AffectsContent and this would affect how grid is rebuilt.

Reply:
See https://github.com/telerik/UI-For-UWP/pull/513#issuecomment-1283460780